src/app/auth/components/LogIn/LogIn.tsx

```diff
@@ -138,12 +138,6 @@ export default function LogIn(): JSX.Element {
         window.rudderanalytics.identify(user.uuid, { email: user.email, uuid: user.uuid });
         window.rudderanalytics.track('User Signin', { email: user.email });
 
-        // analyticsService.identify(user, user.email);
```

Comment:
Why are we removing this?

Reply:
It was commented. Now, I guess we use:
`window.rudderanalytics.identify(user.uuid, { email: user.email, uuid: user.uuid });
        window.rudderanalytics.track('User Signin', { email: user.email });` (line 138/139)